source/server/spongineer/debugging.rst

```diff
@@ -119,3 +119,102 @@ A ``NoClassDefFoundError`` occurs when the plugin tries to access a class that i
+    /*     org.spongepowered.common.event.tracking.TrackingUtil.tickEntity(TrackingUtil.java:160)                                 */
+    /*     net.minecraft.world.WorldServer.redirect$onCallEntityUpdate$zlo000(WorldServer.java:2986)                              */
+    /*     net.minecraft.world.World.func_72866_a(World.java:4154)                                                                */
+    /*     net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)                                                     */
+    /*     net.minecraft.world.World.func_72870_g(World.java:1952)                                                                */
+    /*     net.minecraft.world.World.func_72939_s(World.java:6596)                                                                */
+    /*     net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:2300)                                                    */
+    /*     net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)                                            */
+    /*     net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396)                                  */
+    /*     net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)                                            */
+    /*     net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)                                                     */
+    /*     java.lang.Thread.run(Unknown Source)                                                                                   */
+    /*  Phases Remaining:                                                                                                         */
+    /*                                                                                                                            */
+    /* Minecraft : 1.12.2                                                                                                         */
+    /* SpongeAPI : 7.1.0-SNAPSHOT-7105dfc                                                                                         */
+    /* SpongeForge : 1.12.2-2705-7.1.0-BETA-3361                                                                                  */
+    /* Minecraft Forge : 14.23.4.2705                                                                                             */
+    /******************************************************************************************************************************/
+
+This stacktrace contains the most important version numbers, as well some information about the phase the server was in.
+In this case a ``NullPointerException`` has been thrown during ``EntityTickPhase``. At this point its important to check
+which plugins are involved with the crash. This requires some research as you have to match the package names with mod
+names (Also check the ``Caused by`` blocks).
```

Comment:
It looks like an afterthought (And it should not start with a capital anyway). Hopefully this does it better:
"... names; checking the ``Caused by`` blocks may also help."